### ISSUE TITLE
Missing static file

### ISSUE THREAD

**Issue by rfleschenberg** [jrief/djangocms-cascade#243]
Cascade ships a copy of `bootstrap.min.css`. That file references the file `glyphicons-halflings-regular.eot`, but cascade does not ship that file. This leads to an error when trying to run `collectstatic` with `ManifestStaticFilesStorage` enabled:

```
ValueError: The file 'cascade/css/admin/../fonts/glyphicons-halflings-regular.eot' could not be found with <django.contrib.staticfiles.storage.ManifestStaticFilesStorage object at 0x7f8037e2a470>.
```

The easy solution seems to be to ship the font file. DRF, for example, does the same:

https://github.com/encode/django-rest-framework/tree/master/rest_framework/static/rest_framework/fonts

**Comment by jrief** [comment]
We should get rid of third party JavaScript files anyway and relay solely on ``packages.json`` and ``npm``.

**Comment by rfleschenberg** [comment]
I have to say that I don't really like it if a reusable app tells me which frontend tooling to use, but I guess npm should work.

**Comment by haricot** [comment]
[Grablib](https://github.com/samuelcolvin/grablib)(Static asset management in python) could be an option?

**Comment by jrief** [comment]
@haricot thank for the link on this interesting project.
However, something I'm wondering, how **grablib** handles its own dependencies. Some **npm** packages pull in many other packages they depend on, but I don't see this feature in grablib.

**Comment by racitup** [comment]
Found this error too when trying to use whitenoise to serve static files for heroku.
It also can't find:
```
glyphicons-halflings-regular.woff2
glyphicons-halflings-regular.woff
glyphicons-halflings-regular.ttf
glyphicons-halflings-regular.svg
```

**Comment by jrief** [comment]
**djangocms-cascade** does not ship these files, they are part of the **bootstrap** Node package.
You should install them through ``npm install bootstrap@3`` and point ``STATICFILES_DIRS`` to its folder.

**Comment by racitup** [comment]
As René states though, it looks in `cascade/css/admin/../fonts/` so djangocms-cascade either needs to include both css and fonts, or as you say, find them both somewhere else.
A few times I've wondered why the path to bootstrap isn't configurable since a lot of modules depend on it. I have about 5 copies in my project...

**Comment by jrief** [comment]
which Cascade editor look for the fonts in ``cascade/css/admin/../fonts/``?

**Comment by racitup** [comment]
None, it's where `cascade/css/admin/bootstrap.min.css` looks for the font files:
`url('../fonts/glyphicons-halflings-regular.woff') format('woff')`
whitenoise `CompressedManifestStaticFilesStorage` just checks the dependencies exist and raises an error

**Comment by racitup** [comment]
Thanks, that seems to have done the trick. Although now I've had to use the draft django-SHOP branch I've got another error.. I'll raise another issue.

**Comment by jrief** [comment]
this has been fixed in 0.14.1
